File: CS50/C/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

typedef struct
{
  string name;
  string number;
}
person;

int main (void)
{
  person people[3];

  people[0].name = "carter";
  people[0].number = "+55-11-955187525";

  people[1].name = "Renan";
  people[1].number = "+55-11-965187526";

  people[2].name = "David";
  people[2].number = "+55-11-985187528";

  string name = get_string("Name: ");
  for (int i = 0; i < 3; i++)
  {
    if (strcmp(people[i].name, name) == 0)
    {
      printf("Found %s\n", people[i].number);
      return 0;   
    }
  }
  printf("Not found\n");
  return 1;
}

File: CS50/Python/phonebook.py
  # dictionary
people = {
  "Carter": "+1-617-49511",
  "David": "+1-617-4922",
  "John": "+1-617-4933"
}

name = input("Name: ")

if name in people:
  number = people[name]
  print(f"Found {number}")
else:
  print("Not found")




# names = ["Carter", "David", "John"]

# name = input("Name: ")

# if name in names:
#     print("Found")

# else:
#     print("Not Found")